android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BluetoothSite.java

```diff
@@ -228,7 +228,16 @@ private File getHtmlFile() {
     return new File(websiteDir, "website.html");
   }
 
+  private File getTempFile() {
+    File websiteDir = new File(activity.getFilesDir(), "Websites");
+    websiteDir.mkdir();
+    return new File(websiteDir, "temp.html");
```

Comment:
I added this to the xml/file_path.xml
<cache-path name="cache" path="cachepath" />
and i receive the same error
java.lang.IllegalArgumentException: Failed to find configured root that contains /data/data/physical_web.org.physicalweb/cache/temp-798576872.tmp